### ISSUE TITLE
Resnet50 notebook

### ISSUE THREAD

**Issue by akappeler** [insikk/Grad-CAM-tensorflow#7]
Hi Insik

Firstly, thanks for implementing Grad-Cam.
When I played around with the Resnet50 notebook
I noticed that the prediction results change if you change the order of the images (img1,img2,img3)
Something is not quite right, but I haven't figured out yet where the bug is.

Best
Armin

**Comment by YoungXu06** [comment]
@akappeler Yes, I also find the same problem, when I inference with only one image in a batch, the predicted result is quite wrong. I guess somewhere in the image preprocessing  steps should be responsible for this, but I haven't figure it out yet, do you have any clues?

@insikk Do you have any ideas about this？ THX :D
